funnel/templates/project.html.jinja2

```diff
@@ -69,13 +69,45 @@
     </div>
   </div>
 
-  <div class="grid about-ticket-wrapper">
-    <div class="{% if project.boxoffice_data and project.boxoffice_data.item_collection_id -%}grid__col-sm-6 grid__col-md-7 {% elif project.allow_rsvp -%}grid__col-sm-6{% else %}grid__col-sm-12{% endif %}" id="about">
-      <h2 class="mui--text-display1 mui--text-left project-section__headline">About</h2>
-      <div class="project-section__description">{{ project.description|safe }}</div>
+  <div class="about">
+    {% if project.boxoffice_data and project.boxoffice_data.item_collection_id or project.allow_rsvp -%}
+      <div class="about__participate">
+        <div class="mui--hidden-md mui--hidden-lg mui--hidden-xl about__participate__back-nav">
+          <div class="mobile-nav mui--z1">
+            <a href="javascript(0)" aria-label="Back to {{project.title}}" class="mui--text-dark mui--text-headline" id="close-ticket-widget"><i class="material-icons mui--align-middle mobile-nav__prev" aria-hidden="true">arrow_back</i></a> <span class="mui--text-dark mui--text-headline">Tickets</span>
+          </div>
+        </div>
+        {% if project.boxoffice_data and project.boxoffice_data.item_collection_id -%}
+          <div id="tickets">
+            {% if project.boxoffice_data and project.boxoffice_data.item_collection_id -%}
+              <h2 class="mui--text-display1 mui--text-left project-section__headline">Tickets</h2>
+              <div id="boxoffice-widget"><p class="mui--text-body1">Loading...</p></div>
+            {%- endif %}
+          </div>
+        {% elif project.allow_rsvp -%}
+          <div id="tickets">
+            <h2 class="mui--text-display1 mui--text-left project-section__headline">{% trans %}Attending this event?{% endtrans %}</h2>
+            {%- if g.user %}
+              {{ rsvpform(project, rsvp_form) }}
+            {%- else %}
+              <p>
+                <a href="{{ url_for('login') }}" class='mui-btn mui-btn--raised mui-btn--primary' data-action="Login to rsvp">{% trans %}Login to RSVP{% endtrans %}</a>
+              </p>
+            {%- endif %}
+            {% if project.current_roles.admin or current_auth.permissions.view_rsvps %}
+              <a href="{{ project.url_for('rsvp_list') }}">See RSVP list</a>
+            {% endif %}
+          </div>
+        {%- endif %}
+      </div>
+    {%- endif %}
+
+    <div class="about__details" id="about">
+      <h2 class="mui--text-display1 mui--text-left about__details__headline">About</h2>
+      <div class="about__details__description">{{ project.description|safe }}</div>
```

Comment:
Don't use `|safe`. If it contains HTML, it should be defined as a markup string in Python, not here.